examples/bri-3/README.md

```diff
@@ -68,6 +74,21 @@ $ npm run test
 $ comming soon
 ```
 
+## Trying out NATS messaging
```

Comment:
do we need this in readme if we will have both pub and sub handled in bpi?

Reply:
This is just a temporary entry so that people can test NATS messaging, will be removed in later phases